css/properties/place-content.json

```diff
@@ -2,67 +2,122 @@
   "css": {
     "properties": {
       "place-content": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/place-content",
-          "support": {
-            "webview_android": {
-              "version_added": "59"
-            },
-            "chrome": {
-              "version_added": "59"
-            },
-            "chrome_android": {
-              "version_added": "59"
-            },
-            "edge": {
-              "version_added": false
-            },
-            "edge_mobile": {
-              "version_added": null
-            },
-            "firefox": {
-              "version_added": true
-            },
-            "firefox_android": {
-              "version_added": true
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": true
-            },
-            "safari": {
-              "version_added": true
-            },
-            "safari_ios": {
-              "version_added": null
+        "flex_context": {
+          "__compat": {
+            "description": "Supported in Flex Layout",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/place-content",
+            "support": {
+              "webview_android": {
+                "version_added": "59"
+              },
+              "chrome": {
+                "version_added": "59"
+              },
+              "chrome_android": {
+                "version_added": "59"
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
+                "version_added": "7.0"
+              }
             },
-            "samsunginternet_android": {
-              "version_added": "7.0"
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
             }
           },
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": false
+          "prevent-duplicated-values": {
```

Comment:
It seems like this data is not specific to the layout system in use. In that case, I think it makes sense to have a single `"prevent-duplicated-values"` feature at the same level as `"flex_context"` and `"grid_context"`